tools/data_source/ncbi_datasets.xml

```diff
@@ -5,9 +5,12 @@
     </edam_operations>
     <requirements>
         <requirement type="package" version="12.11.0">ncbi-datasets-cli</requirement>
+        <requirement type="package" version="2021.10.8">ca-certificates</requirement>
```

Comment:
shouldn't this be in the conda package?

Reply:
Yeah, I agree, will open a PR.

Reply:
Actually I already did https://github.com/conda-forge/ncbi-datasets-cli-feedstock/pull/1

Reply:
```suggestion
```